lib/index.js

```diff
@@ -152,7 +152,8 @@ const plugin = options => {
       const json = jsonfile.readFileSync(source_filepath);
 
       // log('File json: %o', json);
-      json.forEach(element => {
+      for (let key in json) {
```

Comment:
Could you add a test within the for loop for `hasOwnProperty` to avoid issues.
See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty#Iterating_over_the_properties_of_an_object

It still allows the use of objects and arrays.